Online/State/RealizedPhysicalObjectState.cs

```diff
@@ -8,6 +8,8 @@ namespace RainMeadow
     [DeltaSupport(level = StateHandler.DeltaSupport.NullableDelta)]
     public class RealizedPhysicalObjectState : OnlineState
     {
+        public float chunkPosThreshold = 0f;
+
```

Comment:
whatever value you put here is at mercy of the incoming flow of state-objects and the state of the delta-encoding stream.

if the incoming object is in full, ie a new room owner or new object owner, then it will be overwritten, if the incoming is delta, then the value will be copied from a baseline.

basically all of this is to say, it's not safe to put it here. this should be in onlinephysicalobject instead

Reply:
we don't rely on persistence, it's just a convenient way to tell the RPOstate chunk reader whether to be more lenient
I considered refactoring readto to take it as an argument but that'd require forwarding it through the chain of base.readto's which is very ugly

Reply:
you'll need a mechanism to set it back to default consistently then, either before of after use, don't rely on the constructor default because the object might be copied from a baseline

Reply:
oops didn't realize we could restore to default after, I'll update the PR